app/views/govuk_publishing_components/components/docs/contents_list.yml

```diff
@@ -181,9 +181,9 @@ examples:
     context:
       right_to_left: true
   with_branding:
```

Comment:
Is it worth renaming these options to `with_no_10_branding`? Since that's the only option for branding now available?